api/controllers/build-task.js

```diff
@@ -69,7 +69,9 @@ module.exports = wrapHandlers({
       return res.forbidden();
     }
 
-    await task.update(body);
+    const updatedTask = { ...body, updatedAt: Date.now() };
```

Comment:
`updatedAt` doesn't automatically get a new timestamp on update?

Reply:
good call, it does, I added this when I didn't see updates but it was a front-end spelling error